src/app/view/components/answersButtons.js

```diff
@@ -0,0 +1,32 @@
+import elementCreator from '../../utils/elementCreator';
+
+/**
+ * At this moment our parameter for this function is question
+ * @param question
+ * question as an object with answers property (answers as array)
+ * this function create buttons and return an array of buttons
+ * @returns {*[]}
+ */
+
+function answersButtons(question) {
+  const buttons = [];
+  question.answers.forEach((button, index) => {
+    buttons.push(
+      elementCreator(
+        'button',
+        { class: 'answerButton' },
```

Comment:
See if you can incorporate BEM into the class naming so button classes are consistent with the rest of the page